ddtrace/sourcecode/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""
+`Datadog Source Code Integration`__ is supported for Git by the addition of the
+repository URL and commit hash in the Python package metadata field
+``Project-URL`` with name ``source_code_link``.
+
+Format of ``source_code_link``: ``<repository url>#<commit hash>``
+
+
+setuptools
+----------
+
+The ``ddtrace`` provides automatic instrumentation of ``setuptools`` to embed
+the source code link into the project metadata. The package has to be installed
```

Comment:
```suggestion
the source code link into the project metadata. ``ddtrace`` has to be installed
```

I think "The package" isn't very clear here.